_layouts/default.html

```diff
@@ -11,7 +11,6 @@
 
     <meta name="format-detection" content="telephone=no">
     <meta name="viewport" content="width=device-width, initial-scale=0.68">
```

Comment:
Revert removal of wanted whitespace.

```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=0.68">

```